automatic/vim/tools/chocolateybeforemodify.ps1

```diff
@@ -0,0 +1,2 @@
+﻿Write-Debug 'Restarting explorer.'
```

Comment:
You restart explorer every time on update while it might not be necessary (particularly given that this is pretty intrusive).